src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,92 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    static final float LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_CAPACITY = 16;
+    private Node<K, V>[] table;
+    private int threshold;
+    private int size;
+
+    MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+    }
+
     @Override
     public void put(K key, V value) {
-
+        ensureTreshold();
+        int index = key == null ? 0 : Math.abs(key.hashCode() % table.length);
+        if (table[index] == null) {
+            table[index] = new Node(key, value, null);
+            size++;
+        } else {
+            Node currentNode = table[index];
+            while (currentNode != null) {
+                if (Objects.equals(key, currentNode.key)) {
+                    currentNode.value = value;
+                    return;
+                }
+                if (currentNode.next == null) {
+                    currentNode.next = new Node(key, value, null);
+                    size++;
+                    return;
+                }
+                currentNode = currentNode.next;
+            }
+        }
     }
 
     @Override
     public V getValue(K key) {
+        int index = key == null ? 0 : Math.abs(key.hashCode() % table.length);
+        Node<K, V> current = table[index];
+        while (current != null) {
+            if (Objects.equals(key, current.key)) {
+                return current.value;
+            } else {
+                current = current.next;
+            }
```

Comment:
You can remove the block else and just put this line after if-block, since you have return in the if-block